### PR TITLE
fix(bug): hostilize Korath mission spawns

### PR DESCRIPTION
**Pull request by TheGiraffe3**
**Bug Fix**

## Summary
Some of the Korath ships from Remnant missions didn't get changed to Kor Alaji, so for example you must make sure nobody is watching the Parca system. You jump in, but the satellites don't deploy and there's nobody hostile.

### CODE REVIEW

**Comment by Zitchas** [comment]
@UnorderedSigh @ravenshining 

Could one or both of you take a look at these to double check the right fleets are being used, since it deals with Korath fleets? Thank you!

**Comment by ravenshining** [comment]
It wouldn't be the Alaji in Parca, it would most likely be the Isaha.  I take it this is happening when you have both played the Korath intro and gotten them to not be hostile, *and* started the Remnant story?  At someplace or another there ought to be a lock out...

Or, for this case, when you take that mission, your rep with the Isaha takes a hit to put you back in the negatives.

It is fortunate I happened to be logged on to check the hud icons PR and saw this in the list, I'm not currently in the habit of checking GitHub notifications.  Best to ping or DM me on discord

(Insert reminder for myself and/or @UnorderedSigh to update the vanilla PR with what we do on this one)

**Comment by TheGiraffe3** [comment]
The pilot that I saw this on hadn't done any missions related to the Korath (hadn't yet discovered Misam) except for during the Remnant storyline.

File: data/remnant/remnant 2 side missions.txt [hunk 1]
```diff
@@ -552,10 +552,10 @@ mission "Remnant: Shattered Light 4"
 		planet "Far Monad"
 		ship "Modified Dromedary Wreck" "Shattered Light"
 	npc evade
-		government "Korath"
+		government "Kor Alaji"
```
Comment (ravenshining) [inline data/remnant/remnant 2 side missions.txt]: Sorry, when I said Parca I meant that whole cluster of systems

```suggestion
		government "Kor Isaha"
```

File: data/remnant/remnant 2 side missions.txt [hunk 2]
```diff
@@ -552,10 +552,10 @@ mission "Remnant: Shattered Light 4"
...
 		personality entering plunders harvests
 		system "Lucina"
-		fleet "Korath Ember Waste Raid"
+		fleet "Kor Alaji Raid"
```
Comment (ravenshining) [inline data/remnant/remnant 2 side missions.txt]:
```suggestion
		fleet "Kor Isaha Raid"
```

File: data/remnant/remnant 2 cognizance.txt [hunk 3]
```diff
@@ -1200,7 +1200,7 @@ mission "Remnant: Cognizance 25"
 				`	"No."`
 					decline
 	npc kill
-		government "Korath"
+		government "Kor Alaji"
```
Comment (ravenshining) [inline data/remnant/remnant 2 cognizance.txt]: These are scouts testing a new prototype and exploring new space, not fanatics attacking the remnant.

```suggestion
		government "Korath"
```

File: data/remnant/remnant 2 cognizance.txt [hunk 4]
```diff
@@ -1200,7 +1200,7 @@ mission "Remnant: Cognizance 25"
 				`	"No."`
 					decline
 	npc kill
```
Comment (ravenshining) [inline data/remnant/remnant 2 cognizance.txt]: If they aren't fanatics, perhaps it is enough just to stop these poor stranded fighters from snooping about, @Zitchas ?

```suggestion
	npc disable
```

File: data/remnant/remnant 2 side missions.txt [hunk 5]
```diff
@@ -675,13 +675,13 @@ mission "Remnant: Scent of Rain 1"
 			`	He nods respectfully. "Well, thank you for all that you have done for us. I will see who else is available." With that, he heads off into the starport.`
 				decline
 	npc "scan outfits"
-		government "Korath"
+		government "Kor Alaji"
```
Comment (ravenshining) [inline data/remnant/remnant 2 side missions.txt]: This is a prototype ship in a Korath system, not someone attacking the Remnant.

```suggestion
		government "Korath"
```

File: data/remnant/remnant 2 side missions.txt [hunk 6]
```diff
@@ -745,13 +745,13 @@ mission "Remnant: Will Not Someone Please Think Of The Children"
 			`	He nods in acknowledgment and his gestures take on a staccato phrasing. "I feared my ask was too much. I understand the sentiment. Perhaps I was naive to think such an outcome was possible."`
 				decline
 	npc disable save
-		government "Korath"
+		government "Kor Alaji"
```
Comment (ravenshining) [inline data/remnant/remnant 2 side missions.txt]: As the mission suggests, this is not a warship.

```suggestion
		government "Korath"
```

File: data/remnant/remnant 2 side missions.txt [hunk 7]
```diff
@@ -745,13 +745,13 @@ mission "Remnant: Will Not Someone Please Think Of The Children"
...
 		personality heroic target marked staying disables plunders
 		system "Vaticanus"
 		ship "Kas'lor Ik 582 (Stranded)" "Fligrookor"
 		dialog `You have disabled the Dathnak A'awoj. Time to report back to <planet>.`
 	npc
-		government "Korath"
+		government "Kor Alaji"
```
Comment (ravenshining) [inline data/remnant/remnant 2 side missions.txt]: Not a warship.

```suggestion
		government "Korath"
```

**Review by ravenshining — CHANGES_REQUESTED** [review]
Now I've had a closer look, I've got some more things to revert / fix.

If the player is fully committed to the Remnant storyline, then normal Korath *should* be hostile.  Generally we try to avoid *needing* to kill factions besides the Alaji, but note that some of these missions are taking place in or near Korath space.  Defending yourself against raiders is one thing, but if you take the fight to them, the Korath will not be pleased.

File: data/remnant/remnant 2 cognizance.txt (hunk 4 above)
Comment (Zitchas) [reply data/remnant/remnant 2 cognizance.txt]: Sure, I can work with that.

**Review by ravenshining — APPROVED** [review]
Looks good.  We should at some point take a deeper look into how hostility is/isn't affected and at what points to cut off mission availability. @UnorderedSigh may know more (and also reminder for both of us to port these changes to the ESC PR)

**Pull request merged**